src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const categoriesSlice = () => (
+  <div>
+    Under Construction;
+  </div>
+);
+
+export default categoriesSlice;
```

Comment:
- [ ] Please update the content of this following the requirements .
- [ ]  In this file,  you should:
`Define a slice of state for categories that:
Includes an array of categories (initial state: empty array)
Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)`

- Please check the requirements 👍🏿